ui/app/components/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react'
```

Comment:
Note that this code is somewhat of a subset of code already in `metamask-extension/ui/app/components/gas-customization/gas-modal-page-container/advanced-tab-content/advanced-tab-content.component.js`

I will probably remove corresponding code from that component in a future task